runners/launch_docker-dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage: launch_docker-dev.sh
+# You can specify LOCAL_MNT and MJKEY_MNT ahead
+
+DOCKER_IMAGE="humancompatibleai/imitation:python-req"
+# Specify LOCAL_MNT if you want to mount a local directory to the docker container
+if [[ ${LOCAL_MNT} == "" ]]; then
+  LOCAL_MNT="${HOME}/imitation"
+fi
+
+# Pass your own mjkey.txt
+if [[ ${MJKEY_MNT} == "" ]]; then
+  MJKEY_MNT="${HOME}/mnt/mjkey.txt"
+fi
+
+# install imitation in developer mode
+CMD="pip install -e .[docs,parallel,test] gym[mujoco]"  # borrowed from ci/build_venv.sh
```

Comment:
One thing I'm confused about: `build_venv.sh` creates a virtual environment `/venv`, so in my version of this script, I activated that. But apparently, `/venv/bin` is already part of `$PATH` for the Docker container, so this might be unnecessary?

Reply:
Yeah I think it's not strictly necessary because the PATH is already set. activate does do some other things (like changing the prompt) but by default nothing else is important I think.